lib/recurly/client.php

```diff
@@ -141,6 +149,14 @@ private function _sendRequest($method, $uri, $data = '')
       curl_setopt($ch, CURLOPT_CUSTOMREQUEST, $method);
     }
 
+    if(Recurly_Client::$logger){
+      Recurly_Client::$logger->info('Send request to Recurly', array('method' => $method, 'uri' => $uri));
+      Recurly_Client::$logger->debug('Send request with headers', array('headers' => $headers));
+      if(!empty($data)){
```

Comment:
I think looking at http://php.net/manual/en/types.comparisons.php I'm thinking `if ($data) {` would be the equivalent.